### ISSUE TITLE
Incorrect serialization of string enums

### ISSUE THREAD

**Issue by gldnspud** [dpranke/pyjson5#46]
This is using Python 3.9.5:

```
from enum import Enum
import json
import json5

class Foo(str, Enum):
    bar = "bar"

x = {"foo": Foo.bar}

print(x)  
# => {'foo': <Foo.bar: 'bar'>}

print(json.dumps(x))  
# => {"foo": "bar"}

print(json5.dumps(x))
# => {foo: ["b", "a", "r"]}

# Temporary workaround:
def json5_dumps(obj):
    return json5.dumps(json.loads(json.dumps(x)))

print(json5_dumps(x))
# => {foo: "bar"}
```

**Comment by dpranke** [comment]
Yup, that looks like a bug, all right. Thanks for reporting it!

**Comment by dpranke** [comment]
Fixed in f1d4ccdcf43a5329917017c37201c81890f6841d and pushed out to PyPI in new version 0.9.6.
